### PR TITLE
Create Largest-Rectangular-Area-histogram.py

### PR DESCRIPTION
**Pull request by hadyshahh**

### CODE REVIEW

File: Largest-Rectangular-Area-histogram.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+	mid = getMid(ss, se);
+	return minVal(hist,RMQUtil(hist,st, ss, mid, qs,
+						qe, 2 * index + 1),
+				RMQUtil(hist,st, mid + 1, se,
+						qs, qe, 2 * index + 2));
+	
+# Return minimum of elements in range
+# from index qs (query start) to
+# qe (query end). It mainly uses RMQUtil()
+
```
Comment (Aniketsonii) [inline Largest-Rectangular-Area-histogram.py]: code missing 
incomplete code 
please complete

**Review by Aniketsonii — CHANGES_REQUESTED** [review]

File: Largest-Rectangular-Area-histogram.py (hunk 1 above)
Comment (hadyshahh) [reply Largest-Rectangular-Area-histogram.py]: okay

**Review by hadyshahh — COMMENTED** [review]
done

**Review by Aniketsonii — APPROVED** [review]
good

**Pull request merged**